jaxtyping/_ipython_extension.py

```diff
@@ -0,0 +1,17 @@
+import IPython
+
+from ._import_hook import _JaxtypingTransformer
+
+
+ip = IPython.get_ipython()
+
+
+def load_ipython_extension(ip):
```

Comment:
Is it possible to add an argument to this? (To specify which typechecker to use.)

Reply:
Sadly, there does not seem to be a naive way. I am currently trying to figure out whether there is a way to pass a parameter without resorting to making global constants-flags.

Reply:
It looks like this is [not implemented](https://github.com/ipython/ipython/issues/6888), but using [this example](https://ipython.readthedocs.io/en/stable/config/extensions/autoreload.html) as inspiration, we could probably do something like
```
%load_ext jaxtyping
%jaxtyping_hook beartype.beartype
```

Reply:
Oh, I saw this only after my last update. Well, this is exactly what I did :smile: